types/hellosign-sdk/hellosign-sdk-tests.ts

```diff
@@ -113,6 +113,7 @@ const signatureRequestResponse: HelloSign.SignatureRequestResponse = {
         details_url: 'details_url',
         cc_email_addresses: ['cc_email_addresses'],
         signing_redirect_url: 'signing_redirect_url',
+	template_ids: ['template-id'],
```

Comment:
<!--
  1 failure:  The root .editorc...
  0 warning: 
  
  
  DangerID: danger-id-Danger;
  File: types/hellosign-sdk/hellosign-sdk-tests.ts;
  Line: 116;
-->
- :no_entry_sign: The root .editorconfig style specifies spaces for whitespace. Please use spaces in new or changed types.

Reply:
Fixed